encryption_works.md

```diff
@@ -337,55 +337,38 @@ Washington Post journalist Barton Gellman, who Edward Snowden trusted with infor
 
-After booting to the DVD you have the option to install Tails on a USB stick, which is especially useful because it allows you to configure a [persistent volume](https://tails.boum.org/doc/first_steps/persistence/index.en.html), an encrypted section of your USB stick to store your data. Despite starting from a clean slate each time you boot up, it's important for you to be able to have access to your OTR and PGP keys, your Claws Mail (more below) and Pidgin settings, and any documents you're working with. Your persistent volume allows you to do this.
+![The Tails operating system.](https://cloud.githubusercontent.com/assets/398046/8490994/b41d71e6-2100-11e5-86f9-6849c216c5e4.png)
 
-### PGP and Email in Tails
+Tails is difficult to use in comparison to a regular operating system. It's slow, and doesn't have all the software you might need. But some of these shortcomings are by design: such limitations make it a lot harder to mess up their endpoint security. If you're in the position where you think the NSA, or another potential attacker, may want to target you or your colleagues (the journalist-whistleblower relationship comes to mind), it's the best tool available. 
 
-I discussed using Thunderbird with the Enigmail add-on to access your email and use PGP, however this software doesn't come with Tails. Tails comes with [Claws Mail](http://www.claws-mail.org/) which includes a PGP plugin.
+Since Tails isn't a practical choice for daily computer use, setting up Tor, PGP, and OTR encryption on your main operating system is a good idea. 
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/claws_mail.png)
+Every time you boot Tails you start from a clean state. Anything and everything you did previously gets erased and the system reverts back to its default out-of-the-box state. It's designed this way so that if you get infected with malware while using Tails, it's gone the next time you boot up. 
 
-Instead of using Enigmail's PGP key management graphical user interface to import, export, generate, view details about, and sign keys, you can click on the clipboard icon in the top right of the screen and choose Manage Keys to open [Seahorse](https://live.gnome.org/Seahorse), which provides these same features.
+To get started, buy a 16 GB USB stick from a store. Then, follow the instructions on [tails.boum.org](https://tails.boum.org/download/index.en.html)'s website. It's an involved process that involves knowledge of the command-line, so you may need to get help from someone with more experience. This is the hardest crypto tool to install, but it is currently the gold standard for anonymously using a  computer, particularly when coupled with PGP-encrypted e-mail with Claws Mail, the e-mail application bundled with Tails. For set-up instructions, check out [Claws](http://www.claws-mail.org/documentation.php?section=general) documentation.
```

Comment:
Also, just a personal nit-pick, but I'd refrain from telling people to go get help from experts.  Although it's very good that you're letting readers know that it's not going to be super-easy, I would encourage them to "stick it out"!